boards/google/quincy/doc/index.rst

```diff
@@ -0,0 +1,47 @@
+.. _google_quincy_board:
+
+Google Quincy Development Board
+###################################
+
+Overview
+********
+
+Quincy is a board created by Google for fingerprint-related functionality
+development.
+
+Board has connectors for fingerprint sensors. Console is exposed over μServo
```

Comment:
Can you add a link to the uServo documentation?

https://chromium.googlesource.com/chromiumos/third_party/hdctools/+/master/docs/servo_micro.md

Reply:
Done, thanks